scalatest/src/main/scala/org/scalatest/tools/TestSortingReporter.scala

```diff
@@ -285,7 +288,8 @@ private[scalatest] class TestSortingReporter(suiteId: String, dispatch: Reporter
     synchronized {
       if (waitingBuffer.size > 0) {
         val head = waitingBuffer.head
-        if (timeoutTask.get.slot.uuid == head.uuid) { // Probably a double check, or just in case there's race condition
+        val (task, _) = timeoutTask.get
```

Comment:
This .get was there before, but I am not convinced the timeoutTask will always be defined. I'll keep looking at it.